qt/scientific_interfaces/Inelastic/Analysis/IndirectFitOutputOptionsView.cpp

```diff
@@ -20,6 +20,8 @@ IndirectFitOutputOptionsView::IndirectFitOutputOptionsView(QWidget *parent)
   connect(m_outputOptions->pbPlot, SIGNAL(clicked()), this, SLOT(emitPlotClicked()));
   connect(m_outputOptions->pbSave, SIGNAL(clicked()), this, SLOT(emitSaveClicked()));
   connect(m_outputOptions->pbEditResult, SIGNAL(clicked()), this, SLOT(emitEditResultClicked()));
+
+  m_outputOptions->pbEditResult->setVisible(false);
```

Comment:
Why this change is introduced?

Reply:
The `Edit Result` option is hidden for the MSDFit and FQFit tabs. It looks like I have forgot to show it for the ConvFit and IqtFit tabs so thanks for spotting this. It should now be fixed in the latest commits